.circleci/config.yml

```diff
@@ -397,6 +406,8 @@ workflows:
               platform: [hirsute, focal, bionic, xenial, centos7, rocky8, bullseye]
       - valgrind:
           <<: *on-integ-and-version-tags
+      - tls:
+          <<: *on-integ-and-version-tags
```

Comment:
Any reason we don't want this on all branches?

Reply:
Juts taking time and I do not think it is needed...